Dockerfile.fedora

```diff
@@ -30,6 +30,9 @@ RUN dnf -y install btrfs-progs-devel \
               container-selinux \
               iptables && dnf clean all
 
+RUN dnf -y upgrade --enablerepo=updates-testing runc container-selinux
+RUN dnf -y install https://copr-be.cloud.fedoraproject.org/results/adrian/criu-podman-test/fedora-28-x86_64/00887063-criu/criu-3.12-0.1.fc28.x86_64.rpm && dnf clean all
+
```

Comment:
Normally we'd want that as a single `RUN` statement and with `dnf clean all` at the end.  However I think this is okay since it's temporary, and the container image isn't used/distributed outside this specific host.

Reply:
This is only a temporary change to see if the CI is happy. Once all the packages I need are part of updates-released I would remove those changes or add them to the main `dnf` command. It is just for testing and better visibility.